textacy/viz/termite.py

```diff
@@ -165,7 +165,8 @@ def draw_termite_plot(
         )
 
         # Create offset transform by 5 points in x direction
-        dx = 10/72.; dy = 0/72.
+        dx = 10 / 72
+        dy = 0
```

Comment:
@bdewilde this is for the angled text. If matplotlib only receives a rotation command, the rotation "anchor" is the corner of the textbox, which includes whitespace. That means, without shifting, the axis tickmarks will be misaligned with the last letters. 

Notice in the example image the tickmarks are directly below the final letters.